colossus/src/main/scala/colossus/protocols/http/HttpResponse.scala

```diff
@@ -10,22 +10,52 @@ import controller._
 import HttpParse._
 
 
-case class HttpResponseHead(version : HttpVersion, code : HttpCode, headers : Vector[(String, String)] = Vector()) extends HttpHeaderUtils {
+case class HttpResponseHeader(key: ByteString, value: ByteString)
+
+object HttpResponseHeader {
+
+  //TODO: these are bytestrings, whereas in Head they're strings
+  val ContentLength = ByteString("content-length")
+  val TransferEncoding = ByteString("transfer-encoding")
+
+  val DELIM = ByteString(": ")
+
+  def apply(key: String, value: String): HttpResponseHeader = {
+    HttpResponseHeader(ByteString(key) , ByteString(value))
+  }
```

Comment:
Good idea but I'm not going to do this quite yet as it'll conflict with another upcoming PR.  Basically we're going to use the same class for both request and response headers, so this code is going to get shifted around anyway.  I'll make sure I add that though.